test/Tests/TestIssues.cpp

```diff
@@ -596,3 +596,30 @@ GAME_TEST(Issues, Issue615) {
     test->playTraceFromTestData("issue_615b.mm7", "issue_615b.json", []() { EXPECT_EQ(pParty->getActiveCharacter(), 1); });
     EXPECT_EQ(pParty->getActiveCharacter(), 4);
 }
+
+GAME_TEST(Issues, Issue625) {
+    // Every character getting club at the start of the game
+    test->loadGameFromTestData("issue_315.mm7");
+    game->goToMainMenu();
```

Comment:
You can drop these two lines & just start the test with `game->pressGuiButton("MainMenu_NewGame");` as all tests start at main menu